include/msgpack/sysdep.h

```diff
@@ -93,7 +93,6 @@ typedef unsigned int _msgpack_atomic_counter_t;
 #endif
 #endif
 
-
```

Comment:
The PR removes many blank lines. However, it does not relate to the fix. Could you keep the blank lines?